apps/assemble.cpp

```diff
@@ -172,8 +172,8 @@ int main(int argc, char** argv)
 
         const char* optname = opt_parms[0];
         const unsigned integration_levels = check_no_adapt(optname,{"-DipSourceMatNoAdapt", "-DSMNA", "-dsmna"}) ? 0 : 10;
-        
-        const Matrix& dsm = DipSourceMat(geo,dipoles,Integrator(3,integration_levels,0.001));
+
+        const Matrix& dsm = DipSourceMat(geo,dipoles,Integrator(3,integration_levels,0.001),domain_name);
```

Comment:
I think this was perhaps a legitimate bug that was hiding behind the default, as the `domain_name` defined above is not used otherwise (other than to emit a message)